blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DAbstractMetricsRW.java

```diff
@@ -50,6 +48,18 @@ protected DAbstractMetricsRW(LocatorIO locatorIO) {
     protected abstract DAbstractMetricIO getIO( String rollupType, Granularity gran );
 
     /**
+     * Returns the Timer.Context object for timing the Datastax write
+     * @return
+     */
+    protected abstract Timer.Context startWriteTimer();
+
+    /**
+     * Returns the Timer.Context object for timing the waits of the write results
+     * @return
+     */
+    protected abstract Timer.Context startWaitResultsTimer();
+
+    /**
```

Comment:
By the way they look, the look kind of similar but there is quite a bit of distinction. `startWriteTimer` is for batch of records whereas `startWaitResultsTimer` is for individual write times. 

Also `startWriteTimer` is not always for a batch of same size. The batch size depends on number of threads available, ROLLUP_BATCH_MIN_SIZE, ROLLUP_BATCH_MAX_SIZE. If the batch size is not consistent, I doubt if this timer would be of any use. Atleast I am concerned if anyone can keep all these into account when looking at graphs.

Overall, I am not a big fan of `startWriteTimer` timer.